plugins/epos/plugin.py

```diff
@@ -139,8 +139,8 @@ def __init__(self, item_id, oai_base=None, lang="en", config=None, name="epos"):
         )
```

Comment:
The config loading is already included in EvalutorBase. Do you think it is better to trust in super() class or we should keep this loading on each plugins?

Reply:
yes, it makes sense to use `super()`. . I'll make the change

Reply:
[Commit 00fa1c89dc0553d91cf468d486ac00f7c278ebed ](https://github.com/IFCA-Advanced-Computing/FAIR_eva/blob/00fa1c89dc0553d91cf468d486ac00f7c278ebed/plugins/epos/plugin.py#L47-L50) allows to pass (*args, **kwargs) from the Plugin class so you don't need to provide the list of input args to `super()`. In case you want to apply it to other plugins.